s3logging acceptance tests fail in TestAccFastlyServiceV1_s3logging_domain_default

Acceptance tests for the `s3logging` sub resource fail due to missing MessageType parameter in the `TestAccFastlyServiceV1_s3logging_domain_default` definition.

```
$ make && TESTARGS=-run=TestAccFastlyServiceV1_s3logging_domain_default make testacc

==> Checking that code complies with gofmt requirements...
go install
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test $(go list ./... |grep -v 'vendor') -v -run=TestAccFastlyServiceV1_s3logging_domain_default -timeout 120m
?       github.com/terraform-providers/terraform-provider-fastly        [no test files]
=== RUN   TestAccFastlyServiceV1_s3logging_domain_default
--- FAIL: TestAccFastlyServiceV1_s3logging_domain_default (8.82s)
        testing.go:428: Step 0 error: Check failed: Check 2/5 error: Bad match S3 logging match, expected (&fastly.S3{ServiceID:"5QwmzXr4nVSyypgPYWCU1G", Version:1, Name:"somebucketlog", BucketName:"fastlytestlogging", Domain:"s3.amazonaws.com", AccessKey:"somekey", SecretKey:"somesecret", Path:"", Period:0xe10, GzipLevel:0x0, Format:"%h %l %u %t %r %>s", FormatVersion:0x1, ResponseCondition:"response_condition_test", MessageType:"", TimestampFormat:"%Y-%m-%dT%H:%M:%S.000", Redundancy:"", CreatedAt:(*time.Time)(nil), UpdatedAt:(*time.Time)(nil), DeletedAt:(*time.Time)(nil)}), got (&fastly.S3{ServiceID:"5QwmzXr4nVSyypgPYWCU1G", Version:1, Name:"somebucketlog", BucketName:"fastlytestlogging", Domain:"s3.amazonaws.com", AccessKey:"somekey", SecretKey:"somesecret", Path:"", Period:0xe10, GzipLevel:0x0, Format:"%h %l %u %t %r %>s", FormatVersion:0x1, ResponseCondition:"response_condition_test", MessageType:"classic", TimestampFormat:"%Y-%m-%dT%H:%M:%S.000", Redundancy:"", CreatedAt:(*time.Time)(nil), UpdatedAt:(*time.Time)(nil), DeletedAt:(*time.Time)(nil)})
FAIL
FAIL    github.com/terraform-providers/terraform-provider-fastly/fastly 8.841s
make: *** [testacc] Error 1
``` 

The following change fixes that specific error.

```diff
diff --git a/fastly/resource_fastly_service_v1_s3logging_test.go b/fastly/resource_fastly_service_v1_s3logging_test.go
index 1956f29..a6d4c84 100644
--- a/fastly/resource_fastly_service_v1_s3logging_test.go
+++ b/fastly/resource_fastly_service_v1_s3logging_test.go
@@ -113,6 +113,7 @@ func TestAccFastlyServiceV1_s3logging_domain_default(t *testing.T) {
                Format:            "%h %l %u %t %r %>s",
                FormatVersion:     1,
                TimestampFormat:   "%Y-%m-%dT%H:%M:%S.000",
+               MessageType:       "classic",
                ResponseCondition: "response_condition_test",
        }
```

However, even if this error is fixed, testing still fails due to hardcoded resource ID in the same test at line 134: https://github.com/terraform-providers/terraform-provider-fastly/blob/master/fastly/resource_fastly_service_v1_s3logging_test.go#L134. The resource key `s3logging.910866243.domain` is not deterministic and would be different for every machine that runs these acceptance tests. 

But, the test is already checking that the domain attribute has `s3.amazonaws.com` in it, isn't it? It's not set in the config function below on line 271, so it must come from the resource defaults.

```
$ make && TESTARGS=-run=TestAccFastlyServiceV1_s3logging_domain_default make testacc

==> Checking that code complies with gofmt requirements...
go install
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test $(go list ./... |grep -v 'vendor') -v -run=TestAccFastlyServiceV1_s3logging_domain_default -timeout 120m
?       github.com/terraform-providers/terraform-provider-fastly        [no test files]
=== RUN   TestAccFastlyServiceV1_s3logging_domain_default
--- FAIL: TestAccFastlyServiceV1_s3logging_domain_default (9.44s)
        testing.go:428: Step 0 error: Check failed: Check 5/5 error: fastly_service_v1.foo: Attribute 's3logging.910866243.domain' not found
FAIL
FAIL    github.com/terraform-providers/terraform-provider-fastly/fastly 9.456s
make: *** [testacc] Error 1
```

--- a/fastly/resource_fastly_service_v1_s3logging_test.go
+++ b/fastly/resource_fastly_service_v1_s3logging_test.go
@@ -113,6 +113,7 @@ func TestAccFastlyServiceV1_s3logging_domain_default(t *testing.T) {
 		Format:            "%h %l %u %t %r %>s",
 		FormatVersion:     1,
 		TimestampFormat:   "%Y-%m-%dT%H:%M:%S.000",
+		MessageType:       "classic",
 		ResponseCondition: "response_condition_test",
 	}
 
@@ -130,8 +131,6 @@ func TestAccFastlyServiceV1_s3logging_domain_default(t *testing.T) {
 						"fastly_service_v1.foo", "name", name),
 					resource.TestCheckResourceAttr(
 						"fastly_service_v1.foo", "s3logging.#", "1"),
-					resource.TestCheckResourceAttr(
-						"fastly_service_v1.foo", "s3logging.910866243.domain", "s3.amazonaws.com"),
 				),
 			},
 		},

